language/move-core/types/src/parser.rs

```diff
@@ -480,6 +485,13 @@ mod tests {
         ] {
             assert!(parse_type_tag(s).is_ok(), "Failed to parse tag {}", s);
         }
+
+        let s = "vector<vector<vector<vector<vector<vector<vector<vector<u64>>>>>>>>";
+        assert!(
+            parse_type_tag(s).is_err(),
+            "Should have failed to parse type tag {}",
+            s
+        );
```

Comment:
Btw there's a test above for "failed" type tags

Reply:
that's transaction arguments